scripts/path_plans.py

```diff
@@ -260,80 +150,217 @@ def _rotationMatrix_orient_z_axis(self, local_point):
         # Rotation Matrix
         # https://en.wikipedia.org/wiki/Rotation_formalisms_in_three_dimensions#Formalism_alternatives
         rot_matrix = np.stack((vector_x, vector_y, vector_z), axis=1)
-        
+
         return rot_matrix
 
 
+class NewInclinedPlane(DetectedObject):
```

Comment:
Since you have deleted the original inclined planes code, I would propose not calling this the 'new' method. I'd just stick with the name `InclinedPlane` as before.

Reply:
Agreed - forgot to change it back after removing the old code 😛

Reply:
Cool. Can you push a commit and resolve when done?